tests/test_django/urls.py

```
@@ -148,11 +203,11 @@
                 "title": Infer,
                 "isbn": Infer,
                 "publication_date": Infer,
-                "publisher_id": Infer,
+                # "publisher_id": Infer,
```

Ruff (0.8.2)

Found commented-out code

Remove commented-out code

(ERA001)